### DISCUSSION TITLE
Dynamically update number field on multiple option picker change

### DISCUSSION THREAD

**Discussion by gerhardmpl** [Budibase/budibase#5394] [Help] [answered]
On one screen, I have some option pickers and - based on the options chosen - different JS calculations will give a number (let's call it _value_). User should be able to try different options to optimize the _value_ but also to override the _value_ manually before saving.

Using a `paragraph component` (with JS in _Text_) it is possible to show an updated value on every option picker change, but user cannot override the calculation by entering a different value. A `number component` (with some JS in _Default value_) does not work either, since it will only update once. 

I could of course use two fields (one to show the dynamically calculated value and another to enter the final value before saving), but would prefer to have only one field for better user experience (especially on mobile devices). Being aware of #5207 I wonder if that is currently possible or planned?

Edit: Ups, just have found #5392 that basically answers this.

**Comment by melohagan** [comment, marked as answer]
Hey @gerhardmpl 

Hopefully this enhancement will be available for the next release. A number of people have requested it, so it'll be nice to have!
